jbpm-system-test/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-parent</artifactId>
+    <version>7.57.0-SNAPSHOT</version>
+  </parent>
+  
+  <groupId>org.jbpm</groupId>
+  <artifactId>jbpm-system-test</artifactId>
+  <version>7.56.0.Final</version>
+  <packaging>pom</packaging>
+
+  <name>jBPM :: System Test</name>
+  <description>jBPM System Test</description>
+
+  <properties>
+    <maven.deploy.skip>true</maven.deploy.skip>
+    <kie.version>7.56.0.Final</kie.version>
+  </properties>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
```

Comment:
it's a trade-off between purity and speed. I agree with you that we should push the most well-polished code but sometimes this requires time to invest, and these playground tests are samples of use. It may be refined in the next iterations.